core/templates/dev/head/editor/feedback_tab/FeedbackTab.js

```diff
@@ -96,19 +96,25 @@ oppia.controller('FeedbackTab', [
       backdrop: true,
       size: 'lg',
       resolve: {
-        suggestion: function() {
-          return $scope.activeThread.suggestion;
+        activeThread: function() {
+          return $scope.activeThread;
         }
       },
       controller: [
-        '$scope', '$modalInstance', 'suggestion',
-        function($scope, $modalInstance, suggestion) {
-        $scope.oldContent = suggestion.state_content.old_content;
-        $scope.newContent = suggestion.state_content.new_content;
+        '$scope', '$modalInstance', 'activeThread', 'explorationData',
+        function($scope, $modalInstance, activeThread, explorationData) {
+        var suggestion = activeThread.suggestion;
+        var states = explorationData.data.states;
+        $scope.oldContent = states[suggestion.state_name].content[0].value;
+        $scope.newContent = suggestion.state_content.value;
+        $scope.commitMessage = {
```

Comment:
Ah, this was actually a workaround because for some reason I put the corresponding view code (the input element for the commit message) in an "ng-if." I've changed the ng-if to ng-show and am just using a string now.